components/Modules/parts/ModuleContextEditor.tsx

```
@@ -0,0 +1,122 @@
+              Cancel
+            </Button>
+            <Button size="sm" onClick={handleContextSave} disabled={isSaving}>
+              Save
+            </Button>
+          </div>
+        </div>
+      ) : (
+        <p
+          className="text-muted-foreground cursor-pointer hover:bg-muted/50 p-4 mr-4 mb-12 rounded min-h-[158px]"
+          onClick={() => setIsEditingContext(true)}
+          tabIndex={0}
```

Biome (1.9.4)

[error] The HTML element p is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)